portfolio/src/main/webapp/styles.css

```diff
@@ -170,3 +170,85 @@ figure {
   background-image: url("../assets/resume.png");
   color: white;
 }
+
+
+.formEmail-container {
+ display: flex;
+ background-color: black;
+ width: 100%;
+ justify-content: space-evenly;
+ align-items: center;
+ height:180px;
+}
+
+.formEmail-p{
+    color: white;
+    margin-bottom: 15px;
+}
+
+.formEmail-textArea{
+    border-radius: 10px;
+    height: 40px;
+    border: solid;
+    padding: 7px 20px 5px 25px;
+    background-image: url("../assets/envelope.jpg");
+    background-size: 15px 15px;
+    background-repeat: no-repeat;
+    background-position-y: center;
+    background-position-x: 5px;
+}
+
+.formEmail-input{
+    background: transparent;
+    color: white;
+    padding: 10px 0;
+    border-radius: 5px;
+    width: 100px;
+    margin-top: 10px;
+    margin-left: 3px;
+}
+
+.formEmail-input:hover{
+    background-color: white;
+    color:black;
+}
+
+/*------------------------------------------------- BucketList */
```

Comment:
I thought I remembered seeing your bucketlist styling in a previous PR? Did you end up not merging that before?

Reply:
Yeah! I did it in a previous PR! :) I think I just set the tag to separate the code.